etc/kayobe/ansible/pulp-image-upload.yml

```diff
@@ -1,21 +1,22 @@
 ---
-- name: Upload and create a distribution for an image
+- name: Upload and create a distribution for an artifact
   hosts: seed
   vars:
     remote_pulp_url: "{{ stackhpc_release_pulp_url }}"
-    remote_pulp_username: "{{ stackhpc_image_repository_username }}"
-    remote_pulp_password: "{{ stackhpc_image_repository_password }}"
-    repository_name: "kayobe-images-{{ openstack_release }}-{{ os_distribution }}-{{ os_release }}"
-    pulp_base_path: "kayobe-images/{{ openstack_release }}/{{ os_distribution }}/{{ os_release }}"
+    remote_pulp_username: "{{ stackhpc_release_pulp_username }}"
+    remote_pulp_password: "{{ stackhpc_release_pulp_password }}"
+    repository_name: "{{ artifact_path }}-{{ openstack_release }}-{{ os_distribution }}-{{ os_release }}"
+    pulp_base_path: "{{ artifact_path }}/{{ openstack_release }}/{{ os_distribution }}/{{ os_release }}"
+    file_extension: 'qcow2'
   tasks:
-    - name: Print image tag
+    - name: Print artifact tag
       debug:
-        msg: "Image tag: {{ host_image_tag }}"
+        msg: "artifact tag: {{ artifact_tag }}"
 
     - name: Get filename
       find:
-        paths: "{{ image_path }}"
-        patterns: '*.qcow2'
+        paths: "{{ artifact_path }}"
+        patterns: "*.{{ file_extension }}"
```

Comment:
Now that we support all sorts of artifacts, this might be better as just a variable for `file_regex` so the user can more tightly control what files are found.